templates/dashboard/order/pdf/invoice.html

```diff
@@ -93,6 +86,9 @@ <h2>{% trans "Items Ordered" context "Invoice ordered items header" %}</h2>
           <td>
             {{ line.product_sku }}
           </td>
+          <td class="right-align">
+            {{ line.unit_price.net|amount:'html' }}
+          </td>
```

Comment:
In France, we are required to show the VAT applied percentage.
- If the VAT percentage doesn't differ from products, we can (can, as we can chose to show on every line if we want to) show the applied VAT percentage in the end of the invoice, with the total tax, total net, and total gross;
- If the VAT percentage does differ from products to products, we must show the applied VAT percentage with the product line.

As we are European countries, I think it would be important to show the applied VAT percentage.

Reply:
I am going to add this soon.